_includes/footer.html

```diff
@@ -1,19 +1,145 @@
-        <a href="{{ '/contact/' | relative_url }}" class="caps h6 white text-decoration-none mr3 {% if page.url == '/contact/' %}bold{% endif %}">{% t meta.contact.title %}</a>
-        <a href="{{ '/policy/' | relative_url }}" class="caps h6 white text-decoration-none {% if page.url == '/policy/' %}bold{% endif %}">{% t meta.privacy_and_security.title %}</a>
+<footer class="footer">
+  <div class="footer-nav">
+    <div class="container">
+      <!-- apply to .footer-nav -->
+      <div class="navigation">
+        <nav>
+          <div class="grid">
+            <!-- apply to nav -->
+            <section>
+              <!-- replace with uls -->
+              <h6>{% t nav.groups.agencies %}</h6>
+              <a class="h6" target="_blank" href="https://partners.login.gov"
+                >{% t nav.become_a_partner %}</a
+              >
+              <a class="h6" target="_blank" href="https://developers.login.gov"
+                >{% t nav.developer_guide %}</a
+              >
+            </section>
+            <section>
+              <h6>{% t nav.groups.learn %}</h6>
+              <a class="h6" href="{{ site.baseurl }}/about/"
+                >{% t nav.about_us %}</a
+              >
+              <a class="h6" href="{{ site.baseurl }}/policy/"
+                >{% t nav.privacy_&_security %}</a
+              >
+              <a
+                class="h6"
+                target="_blank"
+                href="https://design.login.gov/accessibility"
+                >{% t nav.accessibility %}</a
+              >
+            </section>
+            <section>
+              <h6>{% t nav.groups.support %}</h6>
+              <a class="h6" href="{{ site.baseurl }}/help/"
+                >{% t nav.help_center %}</a
+              >
+              <a class="h6" href="{{ site.baseurl }}/contact/"
+                >{% t nav.contact_us %}</a
+              >
+            </section>
+          </div>
+          <div class="system-status">
+            <!-- replace with section footer -->
+            <a class="" target="_blank" href="https://logingov.statuspage.io"
+              >{% t nav.system_status %}</a
+            >
+          </div>
+        </nav>
       </div>
-      <div class="flex flex-auto flex-first">
-        <a href="https://www.gsa.gov" class="flex text-decoration-none white h6" target="_blank">
-          <img src="{{ '/assets/img/square-gsa.svg' | relative_url }}" width="20" height="20" class="mr1 align-middle" alt="GSA hompage" />
-          <span class="sm-show">{% t footer.gsa %}</span>
-        </a>
+    </div>
+  </div>
+  <div class="footer-gsa">
+    <div class="container">
+      <div class="product-mgmt-statement">
+        {% include svg/gsa-logo.svg %} {{
+        site.translations[site.lang]["index"]["footer"]["org"] | replace:
+        'site.baseurl', site.baseurl | markdownify }}
```

Comment:
Do we have a hard line length limit? I think if not this works better with as much of each part on the same line as possible? Maybe like this?

```suggestion
        {% include svg/gsa-logo.svg %}
        {{ site.translations[site.lang]["index"]["footer"]["org"] 
           | replace:'site.baseurl', site.baseurl
           | markdownify }}
```